src/components/Personalization/createNotificationHandler.js

```diff
@@ -24,10 +25,25 @@ export default (collect, renderedPropositions) => {
       return renderedPropositionsDeferred.resolve;
     }
 
-    return (decisionsMeta) => {
-      if (decisionsMeta.length > 0) {
+    return (decisionsMetaDisplay = [], decisionsMetaSuppressed = []) => {
+      if (
+        Array.isArray(decisionsMetaDisplay) &&
```

Comment:
we have an util that does both checks - `isNonEmptyArray`

Reply:
Changed.